### PR TITLE
HPCC-16247 Propagated 'LOG_'datasets to logging

### PR DESCRIPTION
**Pull request by wangkx**
Signed-off-by: wangkx kevin.wang@lexisnexis.com

### CODE REVIEW

**Comment by wangkx** [comment]
@rpastrana please review.

File: esp/logging/loggingagent/espserverloggingagent/loggingagent.cpp [hunk 1]
```diff
@@ -429,6 +436,13 @@ void CESPServerLoggingAgent::filterLogContent(IEspUpdateLogRequestWrap* req)
                 }
                 else if (group == ESPLCGUserReq)
                     originalContentTree.setown(req->getUserRequest());
+                else if (group == ESPLCGLogDatasets)
+                {
+                    const char* logDatasets = req->getLogDatasets();
+                    if (!logDatasets || !*logDatasets)
```
Comment (rpastrana) [inline esp/logging/loggingagent/espserverloggingagent/loggingagent.cpp]: Unless we're within a loop i didn't see, it seems the logic would be clearer like:
if (logDatasets && *logDatasets)
    originalContentTree.setown(createPTreeFromXMLString(logDatasets));
// no need for the continue...

**Review by rpastrana — APPROVED** [review]
@wangkx looks good, only had one stylistic comment.

**Comment by wangkx** [comment]
@richardkchapman this PR is ready to be merged.

**Pull request merged**